src/components/TransactionItemRow/DataCells/CategoryCell.tsx

```diff
@@ -3,21 +3,27 @@ import * as Expensicons from '@components/Icon/Expensicons';
 import TextWithIconCell from '@components/SelectionList/Search/TextWithIconCell';
 import TextWithTooltip from '@components/TextWithTooltip';
 import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
 import type TransactionDataCellProps from './TransactionDataCellProps';
 
 function CategoryCell({shouldUseNarrowLayout, shouldShowTooltip, transactionItem}: TransactionDataCellProps) {
     const styles = useThemeStyles();
+
+    const emptyCategories = CONST.SEARCH.CATEGORY_EMPTY_VALUE.split(',');
+
+    const category = emptyCategories.includes(transactionItem?.category ?? '') ? '' : (transactionItem?.category ?? '');
```

Comment:
I think we can avoid repeating the expression, like

```suggestion
    const category = transactionItem?.category ?? '';
    const categoryForDisplay = emptyCategories.includes(category) ? '' : category;
```